### DISCUSSION TITLE
GitHub Student Pack Application: Rear Camera Issue and Repeated Rejections

### DISCUSSION THREAD

**Discussion by canbedir** [community/community#136577] [GitHub Education]
### Hi everyone,

While applying for the GitHub Student Pack, I encountered an issue with uploading the required document photo using the rear camera. The rear camera option doesn't work, and I've noticed that many others are experiencing the same problem. Taking the document photo with the front camera is difficult, and it's nearly impossible to get a clear image. As a result, my applications have been repeatedly rejected.

Additionally, when answering the question 'Why are you not on campus?', I select the option 'My school hasn't started yet' and upload my class schedule as proof. This should be the correct procedure, right? But it’s really challenging to capture the documents with the front camera...

**Comment by Recycler3** [comment]
Same here.

**Comment by DonsPabloXYZ** [comment]
response #20467

**Reply by DonsPabloXYZ** [reply]
> response #20467
